setup.py

```diff
@@ -13,21 +12,23 @@
     long_description_content_type='text/markdown',
     url='https://github.com/LSSTDESC/SLRealizer',
     install_requires=[
-        "matplotlib >= 2.2.2",
-        "numpy >= 1.13.1",
-        "astropy >= 0.18.1",
-        "om10",
-    ],
-    package_dir={'/lib'},
+        "pybind",
+        "pip==9.0.3",
+        "numpy>=1.13",
+        "future>=0.15",
+        "matplotlib>=2.2.2",
+        "astropy==2.0",
+        "pandas>=0.20",
+        "scikit-learn==0.19.2",
+        "corner",
+        "galsim"],
     include_package_data=True,
```

Comment:
So I guess this is where the setup script goes looking for a file called `MANIFEST.in`, is that right? What was the reason for moving this out of the `setup.py` file?

Reply:
Yes! I believe the `MANIFEST.in` option is more versatile in that it can include/exclude files of any extension inside/outside of a python module. Specifications in `setup.py` are limited to the modules, scripts, and packages required to run the program.